tests/e2e/chain/config.go

```diff
@@ -86,6 +87,7 @@ func addAccount(path, moniker, amountStr string, accAddr sdk.AccAddress) error {
 	balances := banktypes.Balance{Address: accAddr.String(), Coins: coins.Sort()}
 	genAccount := authtypes.NewBaseAccount(accAddr, nil, 0, 0)
 
+	// TODO: Make the SDK make it far cleaner to add an account to GenesisState
```

Comment:
Going to create an issue instead
```suggestion
```

Reply:
https://github.com/osmosis-labs/osmosis/issues/1628

Reply:
I made one on the sdk earlier

Reply:
Also tracked here https://github.com/cosmos/cosmos-sdk/issues/12091